README.md

```diff
@@ -3,9 +3,7 @@ Welcome to docs.bpmn.io. This repository contains documentation for:
 
  - diagram-js
  - bpmn-js
- - dmn-js
-
-libraries.
+ - dmn-js libraries.
```

Comment:
I wouldn't remove the new line here. It's an enumeration, therefore the ending phrase should be below.

Reply:
Ok. I thought it might look better that way. No problem I will make the changes. Any other changes you suggest?